### PR TITLE
Add oc and kubectl download links to docs

### PR DESCRIPTION
**Pull request by agajdosi**
Small docs improvements on experience on Linux.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,8 @@ To login use:
 ### Prerequisites 
 - Currently script prerequisite part only support RHEL/Fedora/CentOS.
 - If you using a different distribution then you need to edit the script around how to install the libvirt packages and make the config changes.
+- [oc 4.0.0](https://mirror.openshift.com/pub/openshift-v3/clients/4.0.0-0.179.0/linux/)
+- [kubectl](https://storage.googleapis.com/kubernetes-release/release/v1.13.3/bin/linux/amd64/kubectl)
```
Comment (LalatenduMohanty) [inline README.md]: I think we should add the link to https://kubernetes.io/docs/tasks/tools/install-kubectl/ here

**Comment by agajdosi** [comment]
@LalatenduMohanty Thanks for suggestion, updated the PR.

File: README.md [hunk 2]
```diff
@@ -100,7 +103,7 @@ $ echo "127.0.0.1 console-openshift-console.apps.test1.tt.testing" | sudo tee -a
 Allow kubectl to bind to privileged ports:
 
 ```
-$ sudo setcap CAP_NET_BIND_SERVICE=+eip /usr/bin/kubectl
+$ sudo setcap CAP_NET_BIND_SERVICE=+eip $(which kubectl)
```
Comment (praveenkumar) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -77,6 +77,9 @@ To login use:
 ### Prerequisites 
 - Currently script prerequisite part only support RHEL/Fedora/CentOS.
 - If you using a different distribution then you need to edit the script around how to install the libvirt packages and make the config changes.
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [oc 4.0.0](https://mirror.openshift.com/pub/openshift-v3/clients/4.0.0-0.179.0/linux/)
```
Comment (praveenkumar) [inline README.md]: 👍

**Pull request merged**